elements/share-tools/README.md

```diff
@@ -0,0 +1,40 @@
+# `<share-tools>`
```

Comment:
Follow a more consistent formatting here, a single large header, and everything else nested.